docs/reference.md

```diff
@@ -587,15 +587,15 @@ Field        | Type    | Description
 amount       | integer | **REQUIRED** The amount of money, in the smallest denomination of the indicated currency.
 currencyCode | string  | The type of currency, in [ISO 4217 format]. For example, the currency code for US dollars is USD.
 
-All `Money` objects require an `amount` and `currencyCode` but `currenyCode` is
+All `Money` objects require an `AmountMoney` and `currencyCode` but `currenyCode` is
 optional for the Reader SDK React Native plugin because `Money` objects will use
 the currency code of the currently authorized location by default.
 
 #### Example JSON
 
 ```json
 {
-  "amount": 100,
+  "AmountMoney": 100,
```

Comment:
This is wrong. it's still "amount", the AmountMoney is the Model type, not the body.

Reply:
Then where we need to change that AmountMoney name ??

Reply:
Done